src/coverlet.core/DependencyInjection.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
```

Comment:
@tonerdo this is a key class, setup a "container" with all injectable services(for now only retry helper).
We have some "static" code(methods) so we need a domain wide model `DependencyInjection.Current` provide access to all services.
I used DI framework used by ASP.NET Code inside `Microsoft.Extensions.DependencyInjection` https://github.com/aspnet/Extensions